### PR TITLE
Fixed CLI name

### PR DESCRIPTION
**Pull request by ShahafBenYakir**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
related: https://github.com/demisto/etc/issues/35366

## Description
The info can be found in the mentioned issue

### CODE REVIEW

**Review by roysagi — APPROVED** [review]

File: Packs/BitSight/IncidentFields/Tags.json [hunk 1]
```diff
@@ -1,12 +1,12 @@
 {
-    "id": "incident_tags",
+    "id": "incident_bitsighttags",
     "version": -1,
     "modified": "2021-02-24T11:46:35.762251019Z",
-    "name": "Tags",
+    "name": "Bitsight Tags",
```
Comment (Itay4) [inline Packs/BitSight/IncidentFields/Tags.json:5]:
```suggestion
    "name": "BitSight Tags",
```

File: Packs/BitSight/Classifiers/classifier-mapper-incoming-BitSight.json [hunk 2]
```diff
@@ -87,7 +87,7 @@
                     },
                     "simple": ""
                 },
-                "Tags": {
+                "Bitsight Tags": {
```
Comment (Itay4) [inline Packs/BitSight/Classifiers/classifier-mapper-incoming-BitSight.json:90]:
```suggestion
                "BitSight Tags": {
```

File: Packs/BitSight/IncidentFields/Tags.json [hunk 3]
```diff
@@ -1,12 +1,12 @@
...
     "ownerOnly": false,
-    "placeholder": "Tags",
-    "description": "Tags",
-    "cliName": "tags",
+    "placeholder": "Bitsight Tags",
```
Comment (Itay4) [inline Packs/BitSight/IncidentFields/Tags.json:7]:
```suggestion
    "placeholder": "BitSight Tags",
```

File: Packs/BitSight/IncidentFields/Tags.json [hunk 4]
```diff
@@ -1,12 +1,12 @@
...
+    "description": "Bitsight Tags",
```
Comment (Itay4) [inline Packs/BitSight/IncidentFields/Tags.json:8]:
```suggestion
    "description": "BitSight Tags",
```

**Review by Itay4 — APPROVED** [review]
please add rn

**Comment by ShahafBenYakir** [comment]
Force merging: as the field name and ids were changed on purpose

**Pull request merged**